tools/changelog.py

```diff
@@ -0,0 +1,276 @@
+"""
+These commands are used manage Salt's changelog.
+"""
+# pylint: disable=resource-leakage,broad-except
+from __future__ import annotations
+
+import datetime
+import logging
+import os
+import pathlib
+import subprocess
+import textwrap
+
+from ptscripts import Context, command_group
+
+log = logging.getLogger(__name__)
+
+REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent
+
+# Define the command group
+cl = command_group(name="changelog", help="Changelog tools", description=__doc__)
+
+
+def changelog(version):
+    """
+    Return a the full changelog generated by towncrier.
```

Comment:
```suggestion
    Return the full changelog generated by towncrier.
```

Reply:
Fixed

Reply:
> How are we going to ensure these updates to the changelog and release notes will be reflected in the master branch?

When we stage a release, we generate a diff file(like we do for regular PR and branch builds).
When we actually release, the release pipeline will apply the diff we make, commit, tag the version, and push everything.